src/__tests__/Raids.test.tsx

```diff
@@ -0,0 +1,131 @@
+/*eslint no-irregular-whitespace: ["error", { "skipTemplates": true }]*/
+import * as React from 'react'
+import { loadingScreen, render, screen, userEvent } from '../tests/testUtils'
+import App from '../App'
+import {
+  fetchedCollectionData,
+  fetchedDocumentData,
+} from '../tests/data/raidsData'
+
+// Note: After each Test Jest doesn't un-mount `The Component` Thats why we don't need to re-visit the page every time
+
+beforeAll(() => {
+  jest.doMock('../utils/useDocument', () =>
+    jest.fn().mockReturnValue(fetchedDocumentData),
+  )
+  jest.doMock('../utils/useCollection', () =>
+    jest.fn().mockReturnValue(fetchedCollectionData),
+  )
+})
+afterAll(() => {
+  jest.resetAllMocks()
+})
+const data = fetchedDocumentData.data
+
+it('should test the ViewRaid $TotalStats', async () => {
+  render(<App />)
+
+  userEvent.click(screen.getByText('Raids'))
+
+  await loadingScreen()
+
+  userEvent.click(
+    screen.getByText(
+      `${fetchedCollectionData.data[0].title} | ${fetchedCollectionData.data[0].dungeon}`,
+    ),
+  )
+  await loadingScreen()
+
+  expect(
+    screen.getByText(`${Object.keys(data.contributors).length} Contributors`),
+  ).toBeInTheDocument()
+  expect(screen.getByText(`${data.commits} Commits`)).toBeInTheDocument()
+  expect(
+    screen.getByText(`${data.changedFiles} Changed Files`),
+  ).toBeInTheDocument()
+  expect(
+    screen.getByText(`+${data.additions} -${data.deletions}`),
+  ).toBeInTheDocument()
+})
+it('should Sort ViewRaid By Addition', async () => {
+  render(<App />)
+
+  const contributorsList = screen.getAllByRole('listitem')
+
+  userEvent.click(screen.getByRole('combobox'))
+  userEvent.click(screen.getByText('additions'))
```

Comment:
Use `userEvent.selectOptions` for this

https://testing-library.com/docs/ecosystem-user-event/#selectoptionselement-values